### PR TITLE
Updating doc with DM open API integration steps

### LINKED ISSUE #78: Add DM openAPI Integration Steps with using External images

### PR DESCRIPTION
**Pull request by anuraggupta228**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #78

### CODE REVIEW

File: EXTERNAL_IMAGES.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ export function decorateMain(main) {
...
 
 Authors can optionally specify query paramaters in the hyperlinked external url and they would be retained in the `picture` tag's `src`/`srcset` attributes. These are useful for specifying image delivery parameters such as image width, height, format, etc. as understood by the external image delivery service.
+
+## Extended Usecase of leveraging DM OpenAPI for Streamlining AEM Franklin Development
+
+For clients utilizing the `AEM Assets` repository to distribute their assets, the burden of maintaining asset integrity due to frequent edits, updates, and deletions performed within AEM is alleviated. Thanks to `Dynamic Media OpenAPI`, synchronization with every DOCX document becomes seamless.
+
+Simply incorporate the necessary `asset-library` configuration into AEM sidekick plugin, as outlined in [here](https://github.com/anuraggupta228/franklinvideo/commit/04d710632f5c614ba7efd52cd9f720f6f4cb58dd)
```
Comment (sdmcraft) [inline EXTERNAL_IMAGES.md:51]: Let's also include the changes in the linked PR to this PR.

File: EXTERNAL_IMAGES.md [hunk 2]
```diff
@@ -40,6 +40,16 @@ export function decorateMain(main) {
...
+
+Once `asset-library` in integrated then simply perform copy/paste through plugin for any asset into target DOCX. Here is one crafted sample on how it might look like - [Check Here](https://docs.google.com/document/d/1fSoGW0X27sZD6KTaZ--7KwNRPBHwO5qjuAnQS1jDa9U/edit?usp=sharing)
```
Comment (sdmcraft) [inline EXTERNAL_IMAGES.md:53]: Same as prior suggestion, let's also have the example doc in this rep's Sharepoint. I think you'd already have access. You can add an example page to demonstrate.

File: EXTERNAL_IMAGES.md [hunk 3]
```diff
@@ -40,6 +40,16 @@ export function decorateMain(main) {
...
+
+And voilà! It’s done! Not only have we minimized redundancy and alleviated the cumbersome task for clients to update every DOCX document with `Dynamic Media Open API` but the LHS still preserves at 100 for mobile as well as desktop - [Witness Here!](https://pagespeed.web.dev/analysis/https-ext-images-with-dm-openapi--franklinvideo--anuraggupta228-hlx-live-dm-open-api-with-aem-franklin/bruylgsqgq?form_factor=mobile)
```
Comment (sdmcraft) [inline EXTERNAL_IMAGES.md:55]: I'd suggest to also add screenshot from https://thinktanked.org/tools/deep-psi/deep-psi.html comparing your page with another page with embedded images.

Comment (anuraggupta228) [reply EXTERNAL_IMAGES.md:55]: done

File: EXTERNAL_IMAGES.md (hunk 2 above)
Comment (anuraggupta228) [reply EXTERNAL_IMAGES.md:53]: added the doc ref as [here](https://adobe.sharepoint.com/:w:/s/HelixProjects/EcC9uxWYlVRCsKzqbjnV44UByeCoBwcRsjz0iPpZgf6zjw?e=Xet9U6) but I cannot make that doc public so have just limited the access to adobe only. 

If you can wish to make it public, can you please do it and share the public ref with me, i'll do the update.

File: EXTERNAL_IMAGES.md (hunk 1 above)
Comment (anuraggupta228) [reply EXTERNAL_IMAGES.md:51]: Done and added at https://github.com/hlxsites/franklin-assets-selector/commit/24e99a2455c9372c8a54637f3e24f0fe4c2ac4f5

File: EXTERNAL_IMAGES.md (hunk 2 above)
Comment (sdmcraft) [reply EXTERNAL_IMAGES.md]: Please place the example word doc in "aem-assets" folder as that's the mountpoint of this repo. No need to add doc link. Instead place the pgae link. Its possible to view the doc source from the page link.

Comment (anuraggupta228) [reply EXTERNAL_IMAGES.md]: done

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**